stage-0-getting-started.md

```diff
@@ -11,6 +11,11 @@ Here are some starting points for learning about the Ethereum protocol.
 
 - Yellow Paper: https://ethereum.github.io/yellowpaper/paper.pdf  (accompanying cheat sheet: [Cheat sheet link](https://github.com/benjaminion/YellowPaper_CheatSheet/blob/master/YPCheatSheet.pdf))
 - Research Forums: https://ethresear.ch/
+- Fellowship of Ethereum Magicians: https://ethereum-magicians.org/
 - EIPS: https://eips.ethereum.org/
 - Write Up on Hexary Trie by ftruzzi: https://gist.github.com/ftruzzi/fe6c6735dbbfafa4994985879254a871
 - Ethereum Glossary of Terms by Piper: https://github.com/ethereum/stateless-ethereum-specs/wiki/Glossary
+
+## Signal your interest
+
+Found a few topics that you are interested in? Signal your interest [here](https://docs.google.com/spreadsheets/d/1xQQ37juWCKrv6F1YDnHX0p4R2HCXlplY3WznZ0m-4fQ/edit?usp=sharing). 
```

Comment:
I appreciate the concern for my sanity, however  I'm ok dishing out merge permissions liberally if the burden gets too high.  If you don't feel strongly about keeping it in the google doc, you've got my blessing to just inline it here and people can submit PRs.  Alternatively, people are welcome to setup their own `./notes/<name>.md` document.

I'll get this merged tomorrow once you've signaled which direction you'd like to take.

Reply:
Having it on gsheets makes it easier to read and perform some simple analytics on the applicants e.g. which is the most sought after topic, which other applicants have similar interests to you etc.

But my biggest concern is having to trust that strangers on the internet will act in good faith and that there are no malicious actors. Even though you can revert changes, it would be troublesome ensure that all legitimate entries are retained in the latest version of the file.

So I think that alone makes the PR approach a much "safer" approach and if you are ok with merging liberally then let's settle with that approach. I will update the PR to reflect this.